httpie/adapters.py

```diff
@@ -0,0 +1,13 @@
+from httpie.cli.dicts import HTTPHeadersDict
+from requests.adapters import HTTPAdapter
+
+
+class HTTPieHTTPAdapter(HTTPAdapter):
+
+    def build_response(self, req, resp):
+        """Wrap the original headers with the `HTTPHeadersDict`
+        to preserve multiple headers that have the same name"""
+
+        response = super().build_response(req, resp)
+        response.headers = HTTPHeadersDict(getattr(resp, "headers", {}))
+        return response
```

Comment:
I am thinking about moving `httpie/ssl.py` into `httpie/adapters.py`, since it does make sense for all the adapters to be in the same place. WDYT @jakubroztocil?

Reply:
Sounds good 👍🏻